DEVELOPER.md

```diff
@@ -133,7 +112,15 @@ cd cloud-platform-deployment-manager
 make && DEBUG=yes make docker-build
 ```
 
+With Golang version 1.17.9, the source directory is not mandatory in ${HOME}/go/src.
+You can create any directory except inside GOPATH and run after the step "git clone" above.
+
 ## Working with a private fork
+With Golang version 1.17.9, go source path is not strictly under
+GOPATH. You can create the working directory except inside GOPATH and clone
```

Comment:
```suggestion
GOPATH. You can create the working directory except inside GOPATH and clone
```
Similar to the other one, I would say:

You can create the working directory anywhere except under GOPATH. and clone your private fork in it.